yarn.lock

```diff
@@ -1647,9 +1647,9 @@
     ethereumjs-util "^5.2.0"
 
 "@cnakazawa/watch@^1.0.3":
```

Comment:
Any idea what this dependency change is from?

Reply:
No, I just did a `yarn` from root and it updated the `yarn.lock` so I believe it was just out of date.